packages/python/genai_prices/__init__.py

```diff
@@ -75,7 +75,7 @@ def extract_usage(
     *,
     provider_id: types.ProviderID | str | None = None,
     provider_api_url: str | None = None,
-    api_flavor: str | None = None,
+    api_flavor: str = 'default',
```

Comment:
There was a mismatch between `api_flavor` type in Python and the definition on the yml schema.

Is this fine, tho?